public/components/metrics/sidebar/metrics_accordion.tsx

```diff
@@ -34,7 +34,7 @@ export const MetricsAccordion = (props: IMetricNameProps) => {
       {metricsList.length > 100 && <p>Use search bar to focus listed Metrics.</p>}
       <ul className="metricsList">
         {metricsList.slice(0, 100).map((metric: any) => (
-          <li key={metric.id} className="metricsListContainer" data-test-subj={dataTestSubj}>
+          <li key={metric?.id} className="metricsListContainer" data-test-subj={dataTestSubj}>
```

Comment:
This is technically a dangerous safe-operator.  We need to understand why the list was iterated with null/undefineds?